### ISSUE TITLE
Modify the loss function

### ISSUE THREAD

**Issue by Gaoang1** [ultralytics/ultralytics#1921]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, I want to use a different loss function for my dataset. But I notice that Yolov8 is imported through ultralytics, does that mean that my changes to the current code won't actually be applied to the training?

**Comment by AyushExel** [comment]
See this example on how to customize elements of the trainer https://docs.ultralytics.com/usage/engine/#customizing-the-detectiontrainer
